lightningd/gossip_control.c

```diff
@@ -107,6 +109,32 @@ static void get_txout(struct subd *gossip, const u8 *msg)
 	}
 }
 
+static void handle_local_channel_update(struct lightningd *ld, const u8 *msg)
+{
+	struct short_channel_id scid;
+	u8 *update;
+	struct channel *channel;
+
+	if (!fromwire_gossipd_got_local_channel_update(msg, msg,
+						       &scid, &update)) {
+		fatal("Gossip gave bad GOSSIP_GOT_LOCAL_CHANNEL_UPDATE %s",
+		      tal_hex(msg, msg));
+	}
+
+	/* In theory this could vanish before gossipd gets around to telling
+	 * us. */
+	channel = any_channel_by_scid(ld, &scid);
+	if (!channel) {
+		log_broken(ld->log, "Local update for bad scid %s",
```

Comment:
Indeed.  It's possible that it happened during the window, but it's very unlikely.

We should add a log_unexpected() which does log_unusual() and includes a magic string "This is unexpected" than catch *that* in CI, rather than using log_broken() for things which may happen (but shouldn't regularly).